frontend/apps/web/yup-validations/connections.ts

```diff
@@ -77,7 +77,7 @@ export const SshTunnelFormValues = Yup.object({
   host: Yup.string(),
   port: Yup.number()
     .min(0)
-    .when('host', (host, schema) => ([host] ? schema.required() : schema)),
+    .when('host', (host, schema) => (host ? schema.required() : schema)),
```

Comment:
Hm, this change seems a little suspect. What changed here that this should not be `[host]` anymore?
I suppose that made it always truthy so it seems like a bug, but how come we didnt catch this before?

Reply:
yeah im not sure - js will always see that as truthy so it wasn't really doing anything which is why i removed it